.gitignore

```diff
@@ -1,2 +1,117 @@
 gwt-material-patterns.iml
 .idea/workspace.xml
+target/gwt-material-patterns-1.0-SNAPSHOT/GwtMaterialPatterns/clear.cache.gif
```

Comment:
Ok will update it

On Mon, Mar 21, 2016, 10:34 PM Ben Dol <notifications@github.com> wrote:

> In .gitignore
> <https://github.com/GwtMaterialDesign/gwt-material-patterns/pull/1#discussion_r56831382>
> :
>
> > @@ -1,2 +1,117 @@
> >  gwt-material-patterns.iml
> >  .idea/workspace.xml
> > +target/gwt-material-patterns-1.0-SNAPSHOT/GwtMaterialPatterns/clear.cache.gif
>
> Use path ignore pattern target/
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/GwtMaterialDesign/gwt-material-patterns/pull/1/files/10dfb0cc7a7faef75c40a2956527df397d9c3358#r56831382>
>